prime-router/src/main/kotlin/cli/SenderUtilsCommand.kt

```diff
@@ -67,27 +72,39 @@ class AddPublicKey : SettingCommand(
             return
         }
         val jwk = SenderUtils.readPublicKeyPemFile(publicKeyFile)
-        jwk.kid = if (kid.isNullOrEmpty()) settingName else kid
+        jwk.kid = if (kid.isNullOrEmpty()) orgName else kid
```

Comment:
Yes, this looks right - from now on the default `kid` is just the orgName.    So, our various examples should just use the `orgName`, and not `orgName.senderName` any more.

Reply:
Ok, I'll find the spots where that's mentioned.

Reply:
Updated the readme for creating keys.